Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
ie:
```diff
diff --git a/Godeps/_workspace/src/github.com/opencontainers/runc b/Godeps/_workspace/src/github.com/opencontainers/runc
new file mode 120000
index 0000000..5930783
--- /dev/null
+++ b/Godeps/_workspace/src/github.com/opencontainers/runc
@@ -0,0 +1 @@
+../../../../..
\ No newline at end of file
diff --git a/Makefile b/Makefile
index 520b012..f7f4acc 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@ RUNC_TEST_IMAGE=runc_test
 PROJECT=github.com/opencontainers/runc
 TEST_DOCKERFILE=script/test_Dockerfile
 BUILDTAGS=seccomp
-export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
+export GOPATH := $(CURDIR)/Godeps/_workspace
 
 all:
        go build -tags "$(BUILDTAGS)" -o runc .
```

Git is happy to track files that you `git add -f` without `.gitignore` changes. :smile: